python/metrics/src/hydrotools/metrics/metrics.py

```diff
@@ -214,6 +214,16 @@ def compute_contingency_table(
         pandas.Series of integer values keyed to pandas.Index([true_positive_key, false_positive_key, false_negative_key, true_negative_key])
         
     """
+    # Check for complete list of categories
+    if True not in observed.cat.categories:
```

Comment:
I'm not super familiar with categorical types...can you no use `any` or something like that? Or is `in` the preferred method of testing?

Reply:
Ugh... this actually made me realize that that line doesn't actually do what I want (and neither will `any`). In this case, we need the behavior of `is`, but `in` works like `==`. I should probably separate this check out to another method and test it.